cmd/qsctl/ls.go

```diff
@@ -127,6 +134,32 @@ func listBucketOutput(s storage.Storager) {
 	fmt.Println(m.Name)
 }
 
+// listBucketLongOutput list buckets with long format
+func listBucketLongOutput(s storage.Storager, t types.SchedulerGetter) {
+	rt := taskutils.NewAtStorageTask(10)
+	rt.SetStorage(s)
+	sst := task.NewStatStorage(rt)
+	t.GetScheduler().Sync(sst)
+	m, err := s.Metadata()
+	if err != nil {
```

Comment:
Any better way to handle this error than just log into debug and ignore?

Reply:
Don't need to handle, just return to user.